packages/viewer/tests/cypress/tests-e2e/3d/layers.cy.ts

```diff
@@ -143,6 +141,8 @@ describe('Test of layer handling in 3D', () => {
         const timeEnabledLayerId = 'test.timeenabled.wmts.layer'
         cy.fixture('layers.fixture.json').then((layersMetadata) => {
             const timedLayerMetadata = layersMetadata[timeEnabledLayerId]
+            // TODO: refactor cypress.d.ts so it gives correct types
+            // @ts-expect-error this is chainable, but cypress type definition says it's a string, so it's causing issues here.
```

Comment:
please fix it while doing this PR, there's already the `cypress.d.ts` file that types any utils we add, you could add the definition here